docs/sources/tutorials/alerting-get-started-pt4/index.md

```diff
@@ -0,0 +1,407 @@
+   - Navigate to **Alerts & IRM > Alerting > Alert rules**.
+   - Click **+ New alert rule**.
+   - Enter an **alert rule name**. High CPU usage
+1. **Define query an alert condition** section:
+
+   - Select TestData data source from the drop-down menu.
+
+     [TestData](https://grafana.com/docs/grafana/latest/datasources/testdata/) is included in the demo environment. If you’re working in Grafana Cloud or your own local Grafana instance, you can add the data source through the Connections menu
+
+   - From **Scenario** select **CSV Content**.
+   - Copy in the following CSV data:
+
+     ```
+     region,cpu-usage,service,instance
+     us-west,88,web-server-1,server-01
+     us-west,81,web-server-1,server-02
+     us-east,79,web-server-2,server-03
+     us-east,52,web-server-2,server-04
+     ```
+
+     This dataset simulates a data source returning multiple time series, with each time series generating a separate alert instance.
+
+1. **Alert condition** section:
+
+   - Keep Last as the value for the reducer function (`WHEN`), and `75` as the threshold value, representing CPU usage above 75% .This is the value above which the alert rule should trigger.
+   - Click **Preview alert rule condition** to run the queries.
+
+   It should return 3 series in Firing state, and 1 in Normal state.
+
+   {{< figure src="/media/docs/alerting/part-4-firing-instances-preview.png" max-width="1200px" caption="Preview of a query returning alert instances" >}}
+
+1. Add folders and labels section:
+
+   - In **Folder**, click **+ New folder** and enter a name. For example: `System metrics` . This folder contains our alert rules.
+
+     Note: while it's possible to template labels here, in this tutorial, we focus on templating the summary and annotations fields instead.
+
+1. **Set evaluation behaviour** section:
+   - In the **Evaluation group and interval**, repeat the above step to create a new evaluation group. Name it `High usage`.
+   - Choose an **Evaluation interval** (how often the alert will be evaluated). Choose `1m`.
+   - Set the **pending period** to 0s (zero seconds), so the alert rule fires the moment the condition is met (this minimizes the waiting time for the demonstration.).
+1. **Configure notifications** section:
+
+   Select who should receive a notification when an alert rule fires.
+
+   - Select a **Contact point**. If you don’t have any contact points, click _View or create contact points_.
+
+1. **Configure notification message** section:
+
+   In this step, you’ll configure the **summary** and **description** annotations to make your alert notifications informative and easy to understand. These annotations use templates to dynamically include key information about the alert.
+
+   - **Summary** annotation: Enter the following code as the value for the annotation.:
+
+     ```go
+     {{ index $labels "instance" }}{{- "\t" -}}{{ index $values "A"}}{{- "\n" -}}
```

Comment:
The screenshot displays additional text:

"Instance:"       server-01. "Usage: 88%"